antea/preproc/io.py

```diff
@@ -0,0 +1,88 @@
+import pandas as pd
+import tables as tb
+import numpy  as np
+import os
+
+from glob import glob
+
+
+def compute_file_chunks_indices(filein, chunk_size = 500000):
+    '''
+    It returns an array with indices separating data in chunks of about 5000000
+    without splitting the data with same event number.
+    '''
+    with tb.open_file(filein) as h5in:
+        evt_numbers = h5in.root.data.cols.evt_number[:]
```

Comment:
Is it necessary to use `[:]`, or could you just drop it?

Reply:
done!